packages/presentation/src/components/markup/NodeContent.svelte

```diff
@@ -49,7 +100,9 @@
   {#if node.type === MarkupNodeType.doc}
     <MarkupNodes {nodes} {preview} />
   {:else if node.type === MarkupNodeType.text}
-    {node.text}
+    <p>
+      {@html urlify(node.text)}
```

Comment:
`@html` is potentially dangerous , we are trying not to use it

Reply:
urlify function, encodes strings to html encoded characters except added links